src/js/pyodide.ts

```diff
@@ -284,6 +293,7 @@ export async function loadPyodide(
     indexURL += "/";
   }
   options.indexURL = indexURL;
+  options.packageCacheDir = options.packageCacheDir ?? indexURL;
```

Comment:
Can you remove this line and put `packageCacheDir: indexURL` into `default_config` below?